src/apple/config/mod.rs

```diff
@@ -188,6 +204,16 @@ impl Config {
             app,
             development_team: raw.development_team,
             project_dir,
+            ios_version: VersionDouble::from_str(
+                &raw.ios_version
+                    .unwrap_or_else(|| DEFAULT_VERSION.to_owned()),
+            )
+            .map_err(Error::IosVersionDoubleInvalid)?,
+            macos_version: VersionDouble::from_str(
+                &raw.macos_version
+                    .unwrap_or_else(|| DEFAULT_VERSION.to_owned()),
```

Comment:
Shouldn't this be a different number on macOS?

Reply:
I didn't really know what to put there exactly as we don't actually use it for anything. 9.0 is a super old version tho, so I think bumping it up to 11 makes sense since I think that's the last still supported version